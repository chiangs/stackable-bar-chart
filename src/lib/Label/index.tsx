import * as React from "react";

type Props = {
  children: any;
};

const NAME_COMPONENT = "stackable-bar-label";

const Label: React.FC<Props> = (props: Props) => (
  <h3 data-testid={NAME_COMPONENT}>{props.children}</h3>
);

export default Label;
