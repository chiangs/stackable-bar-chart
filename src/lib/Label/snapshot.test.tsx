import * as React from "react";
import renderer from "react-test-renderer";
import { describe, expect, test } from "vitest";
import Label from ".";

describe("Label", () => {
  test("Label component renders correctly", () => {
    const component = renderer.create(<Label>Test</Label>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
