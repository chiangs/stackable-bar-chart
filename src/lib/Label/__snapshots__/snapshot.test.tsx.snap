// Vitest Snapshot v1

exports[`Label > Label component renders correctly 1`] = `
<h3
  data-testid="stackable-bar-label"
>
  Test
</h3>
`;
